src/metabase/models/audit_log.clj

```diff
@@ -79,17 +82,17 @@
                  :topic    unqualified-topic
                  :details  details
                  :model    model-name
-                 :model_id model-id)
+                 :model_id model-id
+                 :user_id  user-id)
      ;; TODO: temporarily double-writing to the `activity` table
      (activity/record-activity!
       :topic      topic
       :object     object
       :details-fn (fn [_] details)
-      :user-id    api/*current-user-id*))))
+      :user-id    user-id))))
```

Comment:
Should we default the user-id to current user? Or is it meaningful to have user-id nil in many cases?